packages/desmos-std/src/msg.rs

```diff
@@ -28,35 +41,67 @@ impl Into<CosmosMsg<DesmosMsg>> for DesmosMsg {
 
 impl CustomMsg for DesmosMsg {}
 
+#[cfg(feature = "profiles")]
 impl From<ProfilesMsg> for DesmosMsg {
     fn from(msg: ProfilesMsg) -> Self {
         Self::Profiles(msg)
     }
 }
 
+#[cfg(feature = "subspaces")]
 impl From<SubspacesMsg> for DesmosMsg {
     fn from(msg: SubspacesMsg) -> Self {
         Self::Subspaces(msg)
     }
 }
 
+#[cfg(feature = "relationships")]
+impl From<RelationshipsMsg> for DesmosMsg {
+    fn from(msg: RelationshipsMsg) -> Self {
+        Self::Relationships(msg)
+    }
+}
+
 #[cfg(test)]
 mod tests {
-    use crate::{msg::DesmosMsg, profiles::msg::ProfilesMsg, subspaces::msg::SubspacesMsg};
+    use crate::msg::DesmosMsg;
+
+    #[cfg(feature = "profiles")]
```

Comment:
I think the feature tag could be removed in the test since the everything should be tested.